learn/sherif/main_test/src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -50,7 +50,7 @@ public void setMaxSpeed(double maxSpeed) {
 
     @Override
     public void periodic() {
-        this.differentialDrive.arcadeDrive(this.m_x_speed,this.m_rot, false);
+        this.differentialDrive.arcadeDrive(this.xSpeed,this.rot, false);
     }
 
     public void brake() {
```

Comment:
The brake() method is unnecessary. It would be less code to just use `drive(0, 0);` and that also reduces the possibility of the brake method conflicting with the drive and periodic methods.